spec/pubsub_client/publisher_spec.rb

```diff
@@ -6,30 +6,50 @@ class PubsubClient
 
     let(:topic) { instance_double(Google::Cloud::PubSub::Topic) }
 
-    before do
-      allow(topic)
-        .to receive(:publish_async)
-        .and_yield('the-result')
-    end
-
-    it 'publishes the message asynchronously' do
-      subject.publish('foo') { |_| }
-      expect(topic).to have_received(:publish_async)
-        .with('foo', {})
-    end
-
-    it 'supports attributes' do
-      subject.publish('foo', bar: 'baz') { |_| }
-      expect(topic).to have_received(:publish_async)
-        .with('foo', {bar: 'baz'})
-    end
-
-    it 'yields the result to a block' do
-      yielded_result = nil
-      subject.publish('foo') do |result|
-        yielded_result = result
+    describe 'async publishing' do
+      before do
+        allow(topic)
+          .to receive(:publish_async)
+          .and_yield('the-result')
       end
-      expect(yielded_result).to eq('the-result')
-    end
-  end
+
+      it 'publishes the message' do
+        subject.publish('foo') { |_| }
+        expect(topic).to have_received(:publish_async)
+          .with('foo', {})
+      end
+
+      it 'supports attributes' do
+        subject.publish('foo', bar: 'baz') { |_| }
+        expect(topic).to have_received(:publish_async)
+          .with('foo', {bar: 'baz'})
+      end
+
+      it 'yields the result to a block' do
+        yielded_result = nil
+        subject.publish('foo') do |result|
+          yielded_result = result
+        end
+        expect(yielded_result).to eq('the-result')
+      end
+    end # describe
```

Comment:
Do we need these trailing `describe` comments? Same for line 53, 54. They seem to be debug comments, not sure if it's common to keep them in our ruby codebases or not.

Reply:
I added them because the `end`s can get unwieldly, and it's hard to tell which one aligns with what. I can remove if they are distracting.

Reply:
I think it's fine to keep (I see them in `.ci` files frequently). Was just curious if this was just a typical ruby convention 👍